### ISSUE TITLE
[FEAT] Allow code block rendering

### ISSUE THREAD

**Issue by Quorafind** [marcusolsson/obsidian-projects#39]
I tried to code this already, And it seems like this: ( But finally I failed to add this feature because I don't know how to control `workspaceDef` don't runs into dead loop update. 

![image](https://user-images.githubusercontent.com/13215013/190868705-9d78e51c-7c08-4974-99a0-ee25b1f8179a.png)

![image](https://user-images.githubusercontent.com/13215013/190868698-64c27811-285a-440b-9a0f-b5a198a4cb11.png)

I think it will be a game changer for people who like notion database.

**Comment by marcusolsson** [comment]
I agree that it would be an amazing feature. I probably won't look at this until after 1.0. If anyone wants to give it a go, I'd be happy to provide assistance!

**Comment by Quorafind** [comment]
Nice, I will make my contribution to this and let you review.

**Comment by jebeaudoin** [comment]
This would be a fast-lane route to compatibility with Canvas, right?

**Comment by mjaker** [comment]
Another bump on this feature request.

I unfortunately do not posses the required skills to contribute...

**Comment by Birchyman** [comment]
Also bumping this :)

**Comment by uJeFFF** [comment]
Yes, another bump, I would love to have it
